src/tpm.rs

```diff
@@ -150,10 +150,10 @@ fn write_tpm_data(data: Value) -> Result<(), Box<String>> {
 pub fn is_vtpm() -> Option<bool> {
     match common::STUB_VTPM {
         true => Some(true),
-        false => {
-            let tpm_manufacturer = get_tpm_manufacturer();
-            Some(tpm_manufacturer.unwrap() == "ETHZ")
-        }
+        false => match get_tpm_manufacturer() {
+            Some(data) => Some(data == "EtHZ"),
+            _ => Some(false),
```

Comment:
Why does this function return an `Option<bool>` and not a `bool`?  It never returns `None`...

Reply:
Changed to use boolean instead of Option<>, since bool is sufficient for this function.